etna/models/nn/mlp.py

```diff
@@ -112,7 +112,9 @@ def step(self, batch: MLPBatch, *args, **kwargs):  # type: ignore
     def make_samples(self, df: pd.DataFrame, encoder_length: int, decoder_length: int) -> Iterable[dict]:
         """Make samples from segment DataFrame."""
         values_real = (
-            df.select_dtypes(include=[np.number]).pipe(lambda x: x[[i for i in x.columns if i != "target"]]).values
+            df.drop(["target", "segment", "timestamp"], axis=1).select_dtypes(include=[np.number])
+            # .pipe(lambda x: x[[i for i in x.columns if i != "target"]])
```

Comment:
?

Reply:
I forgot to remove this line. It isn't needed if we drop `target` higher.